ChartsDemo/Swift/DemoBaseViewController.swift

```diff
@@ -117,13 +117,13 @@ class DemoBaseViewController: UIViewController, ChartViewDelegate {
     func handleOption(_ option: Option, forChartView chartView: ChartViewBase) {
         switch option {
         case .toggleValues:
-            for set in chartView.data!.dataSets {
+            for set in chartView.data! {
```

Comment:
I had a feeling that `for set in data` is less natural than `for set in data.dataSets`, what you think? Maybe need time to adapt?
So in the future, we will deprecate `data.dataSet`?

Reply:
I think it's just new to you. We need to continue to support `data.dataSet` as long as we keep Obj-C compatibility.